bindings/bindingtester/spec/bindingApiTester.md

```diff
@@ -277,6 +277,12 @@ futures must apply the following rules to the result:
     internal stack machine state as the last seen version. Pushes the byte
     string "GOT_COMMITTED_VERSION" onto the stack.
 
+#### GET_APPROXIMATE_SIZE
+
+    Calls get_approximate_size and stores the size in the internal stack machine
+    state as the approximate size. Pushes the byte string "GOT_APPROXIMATE_SIZE"
+    onto the stack.
```

Comment:
I think it probably makes sense in this case to change the test to push the string `fdb.tuple.pack((<approximate_size>,))` instead. Doing so would perform more validation that we're getting the right result.

Reply:
Per offline discussion, change to push integer size directly.